### PR TITLE
fix: teleport with  existential deposit

### LINKED ISSUE #8359: Do not allow to teleport less than existential deposit.
### Is there an existing issue?

- [X] I have searched the existing issues

### What happened?

Teleport from polkadothub is successful, but balance not refelcted in polkadot wallet.

I transferred 0.952 dot from polkadothub to polkadot via teleport.
Wallet address used for : 1ZLGLDdV8FHdfKQDmNNW7aneVQ3QBjLpcL9VaULZn5m7qm2

### Please reproduce in steps

open teleport in kodadot
then select polkadothub to polkadot in teleport bridge
select the amount I will transfer (here I transferred 0.952 dot)
then clicked teleport

### Expected Behavior

I think teleport was successful, but fund not reflected in my wallet. Funds are stuck in Polkadot relay.

https://polkadot.subscan.io/xcm_message/polkadot-af2c26e2d54ecc254c13580e5def463f60c83667

### What browsers are you seeing the problem on?

Desktop Chrome

### At which address did you encounter bug?

kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

SubWallet

### At which chain did you encounter bug?

Asset Hub

### Relevant log output

```shell
{
  V3: [
    {
      ReceiveTeleportedAsset: [
        {
          fun: {
            Fungible: "9523818016"
          },
          id: {
            Concrete: {
              interior: {
                Here: "NULL"
              },
              parents: 0
            }
          }
        }
      ]
    },
    {
      ClearOrigin: "NULL"
    },
    {
      BuyExecution: {
        fees: {
          id: {
            Concrete: {
              interior: {
                Here: {
                  set: true
                },
                NULL: {
                  set: false
                }
              },
              parents: 0
            }
          },
          fun: {
            Fungible: "9523818016"
          }
        },
        weight_limit: {
          Unlimited: {
            set: true
          }
        }
      }
    },
    {
      DepositAsset: {
        assets: {
          Wild: {
            AllCounted: 1
          }
        },
        max_assets: 0,
        beneficiary: {
          interior: {
            Here: {
              set: false
            },
            NULL: {
              set: false
            },
            X1: {
              AccountId32: {
                network: null,
                id: "0x18a8b3d4dc1bee79b624359e505a67795e6a4d363683873d66b9a33b3523b216"
              }
            }
          },
          parents: 0
        }
      }
    }
  ]
}
```

### Payment link for reward

1ZLGLDdV8FHdfKQDmNNW7aneVQ3QBjLpcL9VaULZn5m7qm2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #8416: Add existential deposits to static
### Is your feature request related to a problem?

Seems that many issues are happening because of Existential deposits. 

### Describe the solution you would like

In `lib/static`

for each chain create field `deposit: bigint`.

Deposit are as follows:

```csv
chain, deposit
ksm, 333333333
ahk, 33333333
dot, 10000000000
ahp, 1000000000
bsx, <do not know check code>
```

### Describe alternatives you have considered

~

### Screenshots

_No response_

### Are there opened related issues?

_No response_

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix
 
  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #8359
  - [x] Closes #8416

image

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

image

When clicking on the warning text, it would open the url for more details: https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit

  ## Copilot Summary
  
### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 848271d</samp>

This pull request adds a feature to warn users who try to teleport below the existential deposit of the target chain. It uses the `existentialDeposit` object from the `@kodadot1/static` library and adds a translation key and value for the link text.

  
### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 848271d</samp>

> _To teleport funds across chains_
> _You need to avoid certain pains_
> _If you send too low_
> _A warning will show_
> _Or else you might lose all your Plancks_

### CODE REVIEW

File: libs/static/src/chains.ts [hunk 1]
```diff
@@ -86,3 +86,11 @@ export const chainList = (): Option[] => {
     value: prefix,
   }))
 }
+
+export const existentialDeposit: Record<'ksm' | 'ahk' | 'dot' | 'ahp', number> =
```
Comment (vikiival) [inline libs/static/src/chains.ts:90]:
```suggestion
export const existentialDeposit: Record<Prefix, number> =
```

Comment (Jarsen136) [reply libs/static/src/chains.ts]: Updated

**Comment by vikiival** [comment]
Are you checking if this value is less than or equeal to ED?

Screenshot 2023-12-09 at 18 52 30

+ 

> Existential Deposit, there is a risk of potential loss.

`Sending less funds Existential Deposit will result in loss, min value to send is ${existentialDeposit} + fees`

**Comment by Jarsen136** [comment]
> Are you checking if this value is less than or equeal to ED?
> 
> Screenshot 2023-12-09 at 18 52 30
> 

Yes

> 
> > Existential Deposit, there is a risk of potential loss.
> 
> `Sending less funds Existential Deposit will result in loss, min value to send is ${existentialDeposit} + fees`

Will modify the description

**Comment by exezbcz** [comment]
yup, good, lets merge 


followup coming

File: locales/en.json [hunk 2]
```diff
@@ -1084,7 +1084,8 @@
     "congratsSupport": "Congratulations for supporting",
     "tweetDonation": "Tweet about your awesome donation",
     "donationSentTo": "Your donation will be sent to:",
-    "ownerMessage": "(You Are Owner)"
+    "ownerMessage": "(You Are Owner)",
+    "insufficientExistentialDeposit": "Sending less funds Existential Deposit will result in loss, min value to send is [ {0} + fees ] {1}"
```
Comment (prury) [inline locales/en.json:1088]:
```suggestion
    "insufficientExistentialDeposit": "Sending less than the required Existential Deposit amount will result in fund loss, min value to send is [ {0} + fees ] {1}"
```

**Comment by prury** [comment]
tweaked the text a bit
why can't i see the message when trying to send less than ED on kusama?

**Comment by Jarsen136** [comment]
> tweaked the text a bit 

Good. Thank you.

> why can't i see the message when trying to send less than ED on kusama?

image

The value of ED is much less than tx fees on kusama, so you have to input a small value to see the warning message.

**Comment by prury** [comment]
> 
> You can see it here. The value of ED is much less than tx fees on kusama, so you have to input a small value to see the warning message.

oh, right, thank you, i was trying to input a value much more lower than ED

**Review by vikiival — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by JustLuuuu** [comment]
hey guys, there is a mistake in this message? On AssetHub Polkadot existential deposit is 0.1

 Screenshot 2023-12-14 at 14 47 43

Screenshot 2023-12-14 at 14 49 11

**Comment by JustLuuuu** [comment]
Also the message is showing only when you are transferring from AHP to Polkadot

No message when you teleport from Polkadot to AHK, also no messages for Kusama and AHK at all

**Comment by Jarsen136** [comment]
> hey guys, there is a mistake in this message? On AssetHub Polkadot existential deposit is 0.1
> 

It depends on the destination network. In other words, when you teleport money to Polkadot, the ED would be 1 dot.
When you teleport money to Ahp, the ED would be 0.1 dot.

**Comment by roiLeo** [comment]
> hmmm, why we are showing this message? image
> 
>     * its taken from the luu's screenshot

can't reproduce but you should use dot (`.`) instead of comma (`,`)

**Comment by Jarsen136** [comment]
> hmmm, why we are showing this message? 
> * its taken from the luu's screenshot

It's the old version. Now we have redesigned it on another PR.

**Pull request merged**